libs/langchain/langchain/llms/bedrock.py

```diff
@@ -33,6 +34,21 @@ def prepare_input(
 
         return input_body
 
+    @classmethod
+    def prepare_output_stream(
+        self, provider: str, response: Any
```

Comment:
cls not self

Reply:
Fixed. surprised lint didn't pick this one up.